tests/u128.spec.ts

```diff
@@ -1,53 +1,61 @@
 import { expect } from 'chai';
 import { describe, it } from "mocha";
 import * as fs from 'fs';
+import { buildImports } from './utils/helpers';
 
 describe('Basic Operations Test', () => {
     var file = fs.readFileSync("tests/build/u128.wasm");
     var memory = new WebAssembly.Memory({initial: 2});
-    var instance : any;
-
-    var imports = {
-        env: {
-            abort: (msg : string, file: string, line: number, column: number) => {
-                console.error(`abort called at ${ file } (${ line }:${ column })`);
-            },
-            memory: memory
-        }
-    };
+    var buffer = new Uint8Array(memory.buffer);
+    var imports = buildImports("u128.spec.as", memory, buffer);
 
     WebAssembly.instantiate(file, imports)
         .then((result) => {
-            instance = result.instance.exports;
+            let instance = result.instance.exports;
+            it("should add two numbers", () => {
+                expect(instance['BasicOperationsTest#shouldAddTwoNumbers']()).to.be.eq(1);

```

Comment:
It seems this `'BasicOperationsTest#shouldAddTwoNumbers'` can be de-mangled and use more prettify. See: https://github.com/AssemblyScript/assemblyscript/blob/master/lib/loader/index.js#L110